Source/MetroSystem/src/metroSystem/MetroSystem.java

```diff
@@ -29,7 +29,98 @@ public Language getSystemLanguage() {
         return systemLanguage;
     }
 
-    public Database getDatabase() {
-        return database;
+    public Database getDatabase() { return database; }
+
+    /**
+     * Get all edges in a line.
+     * @return  All edges in a line
+     * @since   Nov. 9, 2021
+     */
+    /*
+    public void getAllEdgesInALine(String name, Language language) {
+        Line testLine = null;
+        try {
+            testLine = database.getLineByName(name, language);
+            System.out.println(testLine.getName());
+            ArrayList<Edge> edges = testLine.getEdges();
+            for (Edge e : edges)
+                System.out.println(e.toString());
+            System.out.println();
+        } catch (ExLineNotFound e) {
+            System.out.println(e.getMessage());
+        }
+    }
+    */
+
+    /**
+     * Get all edges in a station.
+     * @return  All edges in a station
+     * @since   Nov. 9, 2021
+     */
+    /*
+    public void getAllEdgesInAStation(String name, Language language, Administrator admin) {
+        Station testStation = null;
+        try {
+            testStation = database.getStationByName(name, language, admin);
+            System.out.println(testStation.getName());
+            ArrayList<Edge> edges = testStation.getEdgeTo();
+            for (Edge e : edges) {
+                System.out.println(e.toString() + ' ' + e.getLine());
+            }
+            System.out.println();
+        } catch (ExStationNotFound e) {
+            System.out.println(e.getMessage());
+        }
+    }
+    */
+
+    public Station getStationByName(String name, Language language, Administrator admin) {
+        try {
+            return database.getStationByName(name, language, admin);
+        } catch (ExStationNotFound e) {
+            System.out.println(e.getMessage());
+            return null;
+        }
+    }
+
+    public Line getLineByName(String name, Language language) {
+        try {
+            return database.getLineByName(name, language);
+        } catch (ExLineNotFound e) {
+            System.out.println(e.getMessage());
+            return null;
+        }
+    }
+
+    /*
+    public void computePrice(Station st_Station, Station ed_Station) {
```

Comment:
Why there are so many codes that are inside the comment block?  
Special usage or forget to delete them?